webapp-django/crashstats/crashstats/static/crashstats/js/socorro/exploitability_report.js

```diff
@@ -0,0 +1,119 @@
+/*global alert:true document:true BugLinks:true */
+(function ($, document) {
+    'use strict';
+
+    function updatePaginationLinks(page, count, size) {
+        var parent = $('.pagination');
+
+        // the "First" link
+        if (page == 1) {
+            // you're on the first page
+            $('a.first', parent)
+                .attr('href', '#1')
+                .addClass('impossible');
+        } else {
+            $('a.first', parent)
+                .attr('href', '#1')
+                .removeClass('impossible');
+        }
+
+        // the "Previous" link
+        if (page > 1) {
+            $('a.previous', parent)
+                .attr('href', '#' + (page - 1))
+                .addClass('possible')
+                .removeClass('impossible');
+        } else {
+            $('a.previous', parent)
+                .attr('href', '#1')
+                .addClass('impossible');
+        }
+
+        // the "Next" link
+        if (page * size < count) {
+            $('a.next', parent)
+                .attr('href', '#' + (page + 1))
+                .removeClass('impossible');
+        } else {
+            $('a.next', parent)
+                .attr('href', '#' + page)
+                .addClass('impossible');
+        }
+
+        // The "Last" link
+        if (page * size < count) {
+            $('a.last', parent)
+                .attr('href', '#' + Math.ceil(count / size))
+                .removeClass('impossible');
+        } else {
+            $('a.last', parent)
+                .attr('href', '#' + Math.ceil(count / size))
+                .addClass('impossible');
+        }
+    }
```

Comment:
I'm really having trouble with this. We have a pagination macro in jinja that works very well and would allow you to do all this in just one line.